portal/src/main/webapp/js/lib/mutationMapper.js

```diff
@@ -1573,18 +1574,26 @@ var PileupUtil = (function()
 				groupCountMap[group] = 0;
 			}
 
-			groupCountMap[group]++;
+			groupCountMap[group] += typeMap[type].length;
 		});
 
 		// convert to array and sort by length (count)
 
 		_.each(_.keys(groupCountMap), function(group) {
-			groupArray.push({group: group, count: groupCountMap[group]});
+			groupArray.push({type: group,
+				count: groupCountMap[group],
+				priority: mutationTypeMap[group].priority});
 		});
 
 		groupArray.sort(function(a, b) {
-			// descending sort
-			return b.count - a.count;
+			if (b.count === a.count) {
+				// tie condition: use mutation type priority
+				return b.priority - a.priority;
```

Comment:
We are setting the priority a few lines above: https://github.com/cBioPortal/cbioportal/pull/1325/files#diff-e64faf74f1b5d548c7cb04ee4bd3aaa2R1585

And the source of this priority is a static map: https://github.com/cBioPortal/mutation-mapper/blob/master/src/js/util/MutationViewsUtil.js#L43

Currently, I don't see a case where the priority can be undefined. But it is always possible that sometime in the future a new entry without a priority value into this static map can break the code :)